src/test/java/api/InstancesApiExamples.java

```diff
@@ -557,7 +563,7 @@ public void canNotUpdateAnExistingMARCInstanceIfBlockedFieldsAreChangedToNulls()
     assertNotNull(putResponse.getJson().getJsonArray("errors"));
     JsonArray errors = putResponse.getJson().getJsonArray("errors");
     assertThat(errors.size(), is(1));
-    assertThat(errors.getString(0), is(
+    assertThat(errors.getJsonObject(0).getString("message"), is(
```

Comment:
This change suggests that the structure of the response has changed, from a string only to an JSON object. Is that the case?

Reply:
Yes, but here used a not standardized validation error response.

Reply:
Is this defined in an interface that is version 1.0 or greater?

Reply:
The error message is returned by `/inventory/instances` endpoint.

Reply:
Does the RAML include a definition for the new structure?

Reply:
No, unfortunately. The RAML does not even have the 422 response code definition.

Reply:
Then I think we should either introduce the 422 response code and switch to using that, or leave the existing structure as is.

Which would you rather do?

(AFAIK FOLIO does not have clear guidance on whether introducing a new kind of response is compatibility breaking)

Reply:
I've introduced the 422 response and incremented inventory API version.